runelite-client/src/main/java/net/runelite/client/ui/components/colorpicker/RuneliteColorPicker.java

```diff
@@ -342,6 +344,15 @@ private void updateHex()
 		updatePanels();
 	}
 
+	public void fitInWindowBounds()
```

Comment:
this doesn't work on monitors that have a different resolution/scaling than main monitor.